sdk/storage/storage-blob/swagger/README.md

```diff
@@ -215,14 +219,6 @@ directive:
       }
 ```
 
-```yaml
```

Comment:
deleted code because it doesn't apply?

Reply:
Yes this was duplicated by mistake, there is an identical transform above